satpy/readers/__init__.py

```diff
@@ -25,6 +24,12 @@
 from datetime import datetime, timedelta
 from functools import total_ordering
 
+try:
+    import fsspec
+    fsspec_module = True
+except ModuleNotFoundError:
+    fsspec_module = False
```

Comment:
Wouldn't this result in `ImportError`?

Also, the `fsspec_module` variable isn't needed, consider:
```suggestion
try:
    import fsspec
except ImportError:
    fsspec = None
```

Reply:
At first I did it this way. I can't remember why I changed it because the suggested way is obviously nicer. Anyway changed it and also added another test for the ImportError case.